design/KruizeDatabaseDesign.md

```diff
@@ -0,0 +1,824 @@
+﻿# **Database Design Document for Kruize**
+
+## **Introduction**
+
+---
+
+The purpose of this document is to describe the database design for the Kruize Autotune project. The Kruize Autotune
+project aims to provide an API to manage experiments by creating,updating and deleting. An api to receive frequent
+metrics results to generate recommendations.
+
+## **Entities**
+
+---
+
+
+The Kruize Autotune project has the following entities:
+
+1. kruize_experiment
```

Comment:
Given that we have the rest of the tables in plural, this should be plural as well
`kruize_experiments`